### PR TITLE
Add the changes in the build system in the changelog

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt [hunk 1]
```diff
@@ -11,7 +11,7 @@ Information to know before starting the upgrade
 Information
 ===========
 
-1. All the files will be formatted with isort, Black and Prettier to be similar than the initial scaffolds
+1. All the files will be formatted with isort, Black and Prettier to be similar than the initial scaffolds.
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt:14]: than -> to

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt [hunk 2]
```diff
@@ -28,6 +28,20 @@ Information
    ``service: geoportal`` by ``service: geoportal-advance``, and in service ``alembic`` you should replace
    ``service: alembic`` by ``service: alembic-advance``.
 
+4. Some little changes in the application build:
+   - We no more use custom build commands to build the docker images, we simply use the docker-compose
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt:32]: We no more -> We no longer

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt [hunk 3]
```diff
@@ -28,6 +28,20 @@ Information
...
+     build command, with tow additional environment variable:
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt:33]: tow -> two
variable -> variables

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt [hunk 4]
```diff
@@ -28,6 +28,20 @@ Information
...
+     * ``SIMPLE``: ``TRUE`` or ``FALSE`` if we compile a simple application or not.
+     * ``GIT_HASH``: the git hash of the current version.
+     Then all the build specification will be moved to the ``docker-compose`` files.
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt:36]: "Then all the build specification will be moved"
->
"The build specification has been moved"

(if I understand correctly)

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt [hunk 5]
```diff
@@ -28,6 +28,20 @@ Information
...
+   - The ``--geoportal`` and ``--config``arguments will be replaced by ``--service=geoportal`` and
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt:37]: will be -> have been

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt [hunk 6]
```diff
@@ -28,6 +28,20 @@ Information
...
+     ``--service=config`` respectively.
+   - The image used by the ``docker-compose`` will be pulled from the Docker hub.
+   - New argument ``--fast-reload`` that will recreate all containers and restart them except the redis once
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt:40]: "the redis once"
we probably mean "the redis containers"? or "the redis ones"
but "the redis containers" is clearer

**Review by jwkaltz — APPROVED** [review]

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt [hunk 7]
```diff
@@ -28,6 +28,20 @@ Information
    ``service: geoportal`` by ``service: geoportal-advance``, and in service ``alembic`` you should replace
    ``service: alembic`` by ``service: alembic-advance``.
 
+4. Some little changes in the application build:
+   - We no longer use custom build commands to build the docker images, we simply use the docker-compose
```
Comment (arnaud-morvan) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt:32]: - In ./build script, we no longer call `docker build` command with custom parameters, but call `docker-compose build` command with wo additional environment variables:

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt [hunk 8]
```diff
@@ -28,6 +28,20 @@ Information
...
+     build command, with two additional environment variables:
+     * ``SIMPLE``: ``TRUE`` or ``FALSE`` if we compile a simple application or not.
+     * ``GIT_HASH``: the git hash of the current version.
+     The build specification has been moved to the ``docker-compose`` files.
```
Comment (arnaud-morvan) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt:36]: Verify rendered position

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt [hunk 9]
```diff
@@ -28,6 +28,20 @@ Information
...
+   - The ``--geoportal`` and ``--config`` arguments have been replaced by ``--service=geoportal`` and
+     ``--service=config`` respectively.
+   - The image used by the ``docker-compose`` will be pulled from the Docker hub.
+   - New argument ``--fast-reload`` that will recreate all containers and restart them except the redis
+     containers to keep the cache.
+   - New argument ``--dry-run``.
```
Comment (arnaud-morvan) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt:42]: ./build script support a new argument ``--dry-run `` which only ... without ...

**Pull request merged**